### ISSUE TITLE
Details about baseline experiments (Table 2)

### ISSUE THREAD

**Issue by MinkyunSeo** [MIT-SPARK/BUFFER-X#10]
In Table 2, we report baseline results (FCGF, PREDATOR, GeoTransformer, and PARENet) under three different experimental settings to analyze generalization behavior under varying levels of prior knowledge and test-time adjustment.

**1. Vanilla**
- Training-time hyperparameters are directly applied to the test dataset without any modification

**2. Oracle**
- Parameters such as voxel size and search radius are adjusted
- Even for the same dataset, the optimal voxel size can vary significantly across methods (e.g., 1.0 for one method vs. 0.3 for another).

**3. Oracle + Scale Align**
- In addition to oracle tuning, the test point clouds are rescaled to match the training-domain scale

For more detail, we provide the pseudo code for the scale alignment procedure used in the *Scale Align* setting.

```python
# -----------------------------------------
# At dataloader (before feeding into model)
# -----------------------------------------
# For example, when trained on 3DMatch (voxel size = 0.025) and tested on KITTI (voxel size = 0.30),
# rescale = 0.025 / 0.30
rescale = train_dataset_voxel_size / test_dataset_voxel_size 

# src_points, tgt_points: (N, 3)
src_points = src_points * rescale
tgt_points = tgt_points * rescale

# -----------------------------------------
# Model inference
# -----------------------------------------
trans_est = model(src_points, tgt_points)

# -----------------------------------------
# Restore scale for evaluation
# -----------------------------------------
trans_est = trans_est / rescale


# (Optional) src_points and tgt_points can also be rescaled back if needed
# src_points = src_points / rescale
# tgt_points = tgt_points / rescale

# Note:
# - Translation error (RTE) must be computed after rescaling back
# - Rotation error (RRE) is invariant to scale and is therefore unaffected
```

### Oracle (Voxel size used)

| Method | ScanNet++i | ScanNet++F | TIERS | KITTI | WOD | KAIST | MIT | ETH | Oxford |
|--------|-----------:|-----------:|------:|------:|----:|------:|----:|----:|-------:|
| FCGF | 0.05 | 0.05 | 0.30 | 0.90 | 0.80 | 1.10 | 1.00 | 0.05 | 0.80 |
| PREDATOR | 0.025 | 0.08 | 0.10 | 0.60 | 0.50 | 1.20 | 0.70 | 0.30 | 0.60 |
| GeoTransformer | 0.025 | 0.10 | 0.20 | 0.70 | 0.70 | 0.90 | 0.40 | 0.50 | 1.50 |
| BUFFER | 0.04 | 0.05 | 0.30 | 0.25 | 0.40 | 0.40 | 0.30 | 0.20 | 0.30 |
| PARENet | 0.025 | 0.10 | 0.25 | 0.80 | 0.70 | 1.20 | 0.40 | 0.50 | 1.30 |

---

### Oracle + Scale Align (Voxel size used)

> If scale-aligned results were worse than oracle, we simply set the rescale factor to **1**, which is equivalent to using the **Oracle-only** setting.
> Such cases are marked with **(*)**.

| Method | ScanNet++i | ScanNet++F | TIERS | KITTI | WOD | KAIST | MIT | ETH | Oxford |
|--------|-----------:|-----------:|------:|------:|----:|------:|----:|----:|-------:|
| FCGF | 0.05 (*) | 0.05 (*) | 0.50 | 0.75 | 0.80 (*) | 1.20 | 1.00 | 0.045 | 0.80 |
| PREDATOR | 0.025 (*) | 0.10 | 0.20 | 0.60 | 0.50 | 1.10 | 0.70 | 0.25 | 0.70 |
| GeoTransformer | 0.04 | 0.10 | 0.25 | 0.80 | 0.80 | 1.10 | 1.10 | 0.35 | 1.50 |
| PARENet | 0.025 | 0.09 | 0.20 | 0.50 | 0.70 | 1.00 | 0.80 | 0.40 | 1.20 |

---

### Notes
- All voxel sizes are reported in meters (m). 
- Values indicate the voxel size used for each method–dataset pair.
- **(*)** denotes cases where scale alignment degraded performance and the rescale factor was set to **1** (Oracle-only).
- BUFFER internally performs scale normalization; therefore, Oracle and Oracle + Scale Align yield identical behavior.

**Comment by qinzheng93** [comment]
Hi, can you provide the oracle voxel size for each dataset?

**Comment by MinkyunSeo** [comment]
First of all, thank you for your support in enabling the generalization experiments during the preparation of our paper.
I learned a lot from studying GeoTransformer, which has been highly influential in shaping my understanding of point cloud registration.

We have added a table above that explicitly reports the oracle voxel sizes used for each method–dataset pair (see “Voxel Size Used in Table 2”).
Please feel free to let us know if you need any additional details or clarifications.
